service/src/main/java/org/kiwiproject/champagne/dao/DeployableSystemDao.java

```diff
@@ -47,9 +47,6 @@ public interface DeployableSystemDao {
     @SqlQuery("select system_admin from users_deployable_systems where user_id = :userId and deployable_system_id = :systemId")
     boolean isUserAdminOfSystem(@Bind("userId") long userId, @Bind("systemId") long systemId);
 
-    @SqlQuery("select true from users_deployable_systems uds join users u on u.id = uds.user_id join deployable_systems ds on ds.id = uds.deployable_system_id where u.system_identifier = :userName and ds.id = :systemId")
```

Comment:
What was this used for?

Reply:
It was used to check if a user was in a system based on their username not just their id. Not needed anymore